src/components/registerV2/backupPassphrase.js

```diff
@@ -120,7 +128,8 @@ class BackupPassphrase extends React.Component {
             </SecondaryButtonV2>
           </span>
           <span className={`${registerStyles.button} ${grid['col-xs-4']}`}>
-            <PrimaryButtonV2>
+            <PrimaryButtonV2 disabled={!this.state.hasSaved}
```

Comment:
This button should be always enabled. The user can just write the passphrase down on a piece of paper instead of copying it.